### PR TITLE
Prepare 1.0.2 release

### PR DESCRIPTION
**Pull request by swissspidy**
Let's wait for

* #496
* #509

### CODE REVIEW

File: includes/Checker/Checks/Plugin_Readme_Check.php [hunk 1]
```diff
@@ -292,7 +292,7 @@ private function check_license( Check_Result $result, string $readme_file, Parse
 	 * Normalice licenses to compare them.
 	 *
 	 * @param string $license The license to normalice.
```
Comment (mukeshpanchal27) [inline includes/Checker/Checks/Plugin_Readme_Check.php]:
```suggestion
```

File: includes/Checker/Checks/Plugin_Readme_Check.php [hunk 2]
```diff
@@ -292,7 +292,7 @@ private function check_license( Check_Result $result, string $readme_file, Parse
...
-	 * @since 1.1.0
+	 * @since 1.0.2
 	 *
 	 * @return string
```
Comment (mukeshpanchal27) [inline includes/Checker/Checks/Plugin_Readme_Check.php:297]:
```suggestion
	 * @param string $license The license to normalice.
	 * @return string
```

File: includes/Checker/Default_Check_Repository.php [hunk 3]
```diff
@@ -54,6 +54,7 @@ private function register_default_checks() {
 				'enqueued_resources'         => new Checks\Enqueued_Resources_Check(),
 				'plugin_readme'              => new Checks\Plugin_Readme_Check(),
 				'enqueued_styles_scope'      => new Checks\Enqueued_Styles_Scope_Check(),
+				'enqueued_scripts_scope'     => new Checks\Enqueued_Scripts_Scope_Check(),
```
Comment (mukeshpanchal27) [inline includes/Checker/Default_Check_Repository.php:57]: Nice catch 👍

**Comment by ernilambar** [comment]
This PR https://github.com/WordPress/plugin-check/pull/490 is also almost ready. May be we could accommodate this also in this release.

**Review by ernilambar — APPROVED** [review]

**Review by mukeshpanchal27 — APPROVED** [review]

**Pull request merged**